source/_layouts/page-pattern.html

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+---
+<div id="main" class="col-md-12 clearfix" role="main">
+  <article class="post">
+    <header class="page-tabs-header">
+      <div class="row">
+        <div class="col-sm-12 col-md-6 cols-nopad-right">
+          <h1>{{ page.title }}</h1>
+        </div>
+        <div class="col-sm-12 col-md-6 cols-nopad-left cols-nopad-right">
+          {% if page.code == false %}
+            {% include nav-tabs-pattern.html codetab=false %}
+          {% elsif page.design == false %}
+            {% include nav-tabs-pattern.html designtab=false %}
+          {% else %}
+            {% include nav-tabs-pattern.html %}
+          {% endif %}
+        </div>
+      </div>
+    </header>
+    <section class="post_content">
+      <div class="tab-content">
+        <div role="tabpanel" class="tab-pane active" id="overview">
```

Comment:
I'm more curious if this was an intentional decision that affects the entire site (i.e., to no longer include examples in the overview)? Or, perhaps that was an over-site when including the heat map overview page in the design repo?

If that's the plan for future overview pages, I'm fine with that. Essentially, our website pages now live in only in the deign repo.

It would be good to know that so we stop adding examples to overview pages. I recently merged a couple last sprint.